src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java

```diff
@@ -102,6 +111,11 @@ public EntryCreatedRS createProjectIntegration(String projectName, IntegrationRQ
 				this.basicIntegrationService
 		);
 
+		ofNullable(createRequest.getName()).map(String::toLowerCase).ifPresent(name -> {
```

Comment:
should we do something if name is null ?

Reply:
validation added